pkg/collector/corechecks/cluster/ksm/kubernetes_state_transformers.go

```diff
@@ -426,7 +428,7 @@ func validateJob(val float64, tags []string) ([]string, bool) {
 	kubeCronjob := ""
 	for _, tag := range tags {
 		split := strings.Split(tag, ":")
-		if len(split) == 2 && split[0] == "kube_job" || split[0] == "job" || split[0] == "job_name" {
+		if len(split) == 2 && split[0] == "kube_job" || split[0] == "job" || split[0] == "job_name" || split[0] == "reason" {
 			// Trim the timestamp suffix to avoid high cardinality
 			if name, trimmed := trimJobTag(split[1]); trimmed {
```

Comment:
These 3 lines seem to expect a colon-separated tag whose key is `reason` to have a value which would be a job name.